framework/contracts/native/module-factory/src/commands.rs

```diff
@@ -29,14 +29,21 @@ pub fn execute_create_modules(
     let block_height = env.block.height;
     // Verify sender is active Account manager
     // Construct feature object to access registry functions
-    let binding = VersionControlContract::new(config.version_control_address);
+    let version_control = VersionControlContract::new(config.version_control_address);
 
-    let version_registry = binding.module_registry(deps.as_ref());
-    let account_registry = binding.account_registry(deps.as_ref());
+    let version_registry = version_control.module_registry(deps.as_ref());
+    let account_registry = version_control.account_registry(deps.as_ref());
 
     // assert that sender is manager
     let account_base = account_registry.assert_manager(&info.sender)?;
 
+    // App base message
+    let app_base_msg = abstract_core::app::BaseInstantiateMsg {
```

Comment:
We only need to construct this when instantiating an app so this can be moved down.